components/nimbus/src/stateless/cirrus_client.rs

```diff
@@ -88,6 +89,10 @@ impl CirrusClient {
         let app_context: AppContext = serde_json::from_str(&app_context)?;
         Ok(Self {
             app_context,
+            // As it currently stands, with no coenrolling feature ids, cirrus cannot
+            // support coenrolling features.
+            // This is the subject of https://mozilla-hub.atlassian.net/browse/EXP-3623
+            coenrolling_feature_ids: Default::default(),
```

Comment:
Hmmmm this is where I'm now kinda wishing the FML library was available here because I'm not sure how we're gonna handle this from a deployment perspective

Reply:
What do you mean? 🤔

Reply:
TL;DR I was wrong, it works fine we just need to add a way to pull the coenrolling feature ids from the FML client

So what I was referring to is that Nimbus will get these values because the generated code has them effectively hard-coded in it and will pass them into the NimbusClient object during instantiation. 

Cirrus does not have the luxury of generated code, so we'd need an environment variable with a comma-separated list or a config file or something of the sort to contain the list of ids. That lends to more chance for human error, so I was noting that it'd be nice if we could import the FML so that we had the FML client object here.

But in writing that out, I realized we don't need that. We just need to update the FML Client to have a method that returns the coenrolling feature ids — then the Python code would be adjusted to look something like this
1. have the FML file mounted somewhere (current plan)
2. instantiate the FML client(s) BEFORE the SDK client(s)
3. get the coenrolling feature ids from the FML client(s) via an FFI'd method
4. instantiate the SDK client(s) passing in the feature ids obtained in the previous step

Reply:
Please see [EXP-3624](https://mozilla-hub.atlassian.net/browse/EXP-3624) and [EXP-3626](https://mozilla-hub.atlassian.net/browse/EXP-3626) 😉 .